modules/kmm-using-driver-toolkit.adoc

```diff
@@ -16,7 +16,7 @@ Use DTK as the first stage of a multi-stage `Dockerfile`.
 
 . Build the kernel modules.
 
-. Copy the `.ko` files into a smaller end-user image such as https://catalog.redhat.com/software/containers/ubi8/ubi-minimal[`ubi-minimal`].
+. Copy the `.ko` files into a smaller end-user image such as https://catalog.redhat.com/software/containers/ubi9/ubi-minimal[`ubi-minimal`].
```

Comment:
@sheriff-rh I couldn't get this link to resolve, but it might be my problem and not the doc problem. The URL works fine  up to the ubi8 or ubi9 part.

Reply:
Yep you're not going to get the URL to work, we are references a universal base image (UBI) not a web page.